src/schema/tags.ts

```diff
@@ -20,6 +20,10 @@ type GQLTagResults = Pick<GQLTagSearchResults, 'hits'>;
 
 export const RECOMMENDED_TAGS_LIMIT = 5;
 
+export const MIN_SEARCH_QUERY_LENGTH = 2;
```

Comment:
Should we not handle this from the FE to avoid even sending the query?

Reply:
Agree

Reply:
I looked at it, I can do the adjustments but not sure it will be intuitive to not enable the query until 2 chars. We might need some label/error state.

Reply:
With enabling query after 2 chars it would work like:
- write 1 char
- nothing happens and you see all tags
- write 2. char
- then the query goes

Reply:
Probably doesn't hurt to have it both places, in case front end doesn't actually prevent it.

Reply:
No matter the frontend I think it is valid to protect the query from manual requests with less then 2 char.

Reply:
Yes, it wasn't to mention we don't need it here just to clear. Just to not forget adjusting the frontend

Reply:
100% always nice as a safety fallback, but if possible let's try and block the FE as well to avoid queries

Reply:
https://github.com/dailydotdev/apps/pull/2286/commits/a940df152f2894835fa90e7144d1b5fbf3ff886d